src/legacy/core_plugins/table_vis/public/shim/legacy_dependencies_plugin.ts

```diff
@@ -17,4 +17,25 @@
  * under the License.
  */
 
-import './agg_table';
+import chrome from 'ui/chrome';
+// @ts-ignore
+import { VisFactoryProvider } from 'ui/vis/vis_factory';
+import { CoreStart, Plugin } from 'kibana/public';
+import { initTableVisLegacyModule } from './table_vis_legacy_module';
+
+export class LegacyDependenciesPlugin implements Plugin<any, any> {
+  public async setup() {
+    initTableVisLegacyModule();
+
+    const $injector = await chrome.dangerouslyGetActiveInjector();
+    const Private = $injector.get('Private');
+
+    return {
+      createAngularVisualization: VisFactoryProvider(Private).createAngularVisualization,
```

Comment:
I like that you didn't end up exposing raw Angular deps here.
I wonder if this is the case for all visualizations.
But here it works great @streamich